mappings/none/bzq.mapping

```diff
@@ -1,5 +1,9 @@
 CLASS none/bzq
+	METHOD a useAmbientOcclusion ()Z
```

Comment:
`usesAmbientOcclusion` IMO so then the call is something like `if (gui.usesAmbientOcclusion())`